xla/service/gpu/ir_emitter_unnested.cc

```diff
@@ -1089,8 +1078,7 @@ absl::Status IrEmitterUnnested::EmitFusedMHABackwardThunk(
     mask_shape = instr->operand(input_index++)->shape();
   }
 
-  bool has_bias = is_flash_attention &&
-                  (kind == CudnnfMHAKind::kBackwardScaleBiasSoftmax ||
+  bool has_bias = (kind == CudnnfMHAKind::kBackwardScaleBiasSoftmax ||
                    kind == CudnnfMHAKind::kBackwardScaleBiasSoftmaxDropout ||
                    kind == CudnnfMHAKind::kBackwardScaleBiasMaskSoftmax ||
```

Comment:
do we still need to keep those CudnnfMHAKinds with mask?

Reply:
Thanks, I forgot to remove this.